application/F3DOptionsParser.cxx

```diff
@@ -311,6 +311,7 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
     this->DeclareOption(grp0, "dry-run", "", "Do not read the configuration file", appOptions.DryRun,  HasDefault::YES, MayHaveConfig::NO );
     this->DeclareOption(grp0, "no-render", "", "Do not render anything and quit right after loading the first file, use with --verbose to recover information about a file.", appOptions.NoRender,  HasDefault::YES, MayHaveConfig::YES );
     this->DeclareOption(grp0, "max-size", "", "Maximum size in Mib of a file to load, negative value means unlimited", appOptions.MaxSize,  HasDefault::YES, MayHaveConfig::YES, "<size in Mib>");
+    this->DeclareOption(grp0, "auto-reload", "", "Reload currently loaded file if it is modified on disk", appOptions.AutoReload,  HasDefault::YES, MayHaveConfig::YES );
```

Comment:
It looks like "file watching" is the accepted terminology and `--watch` is already a thing out there ([`webpack`](https://webpack.js.org/api/cli/), [`tsc`](https://www.typescriptlang.org/docs/handbook/compiler-options.html), [`spawning(1)`](https://linux.die.net/man/1/spawning), [`qiv`](https://linux.die.net/man/1/qiv), [`kubectl`](https://kubernetes.io/docs/reference/kubectl/), ...)

For the description I'm not 100% but maybe using "whenever" instead of "if" is enough to convey the automatic/continuous/repeating idea?
```suggestion
    this->DeclareOption(grp0, "watch", "", "Reload the current file whenever it is modified on disk", appOptions.AutoReload,  HasDefault::YES, MayHaveConfig::YES );
```
also "currently loaded file" vs "current file" vs "active file"? idk